src/plugin/ZeroDevSessionKeyPlugin.sol

```diff
@@ -52,7 +56,9 @@ contract ZeroDevSessionKeyPlugin is ZeroDevBasePlugin {
         bytes calldata signature
     ) internal view override returns (bool) {
         address sessionKey = address(bytes20(data[0:20]));
-        require(!getPolicyStorage().revoked[sessionKey], "session key revoked");
+        if (getPolicyStorage().revoked[sessionKey]) {
```

Comment:
Session key is where i'm most against changing to custom error in my experience, it's really confusing to debug what's going on if all errors return `AA23 reverted (or OOG)`